docker/Dockerfile

```diff
@@ -41,34 +38,30 @@ ENV PIP_USE_PEP517=1
 FROM python-base AS pyproject-builder
 
 # Install dependencies
-RUN \
-  --mount=type=cache,target=/var/cache/apt,sharing=locked \
-  --mount=type=cache,target=/var/lib/apt,sharing=locked \
-  apt-get update \
+RUN apt-get update \
   && apt-get install -y \
     --no-install-recommends \
     build-essential=12.9 \
     gcc=4:10.2.* \
-    python3-dev=3.9.*
+    python3-dev>=${PYTHON_VERSION}
 
 # prepare pip for buildkit cache
+ARG APPNAME=InvokeAI
 ARG PIP_CACHE_DIR=/var/cache/buildkit/pip
 ENV PIP_CACHE_DIR ${PIP_CACHE_DIR}
 RUN mkdir -p ${PIP_CACHE_DIR}
 
 # create virtual environment
-RUN --mount=type=cache,target=${PIP_CACHE_DIR},sharing=locked \
-  python3 -m venv "${APPNAME}" \
+RUN python3 -m venv "${APPNAME}" \
   --upgrade-deps
 
 # copy sources
-COPY --link . .
```

Comment:
why do you remove the --link?

Reply:
See [f70fb02](https://github.com/invoke-ai/InvokeAI/pull/2722/commits/f70fb0278c8d460bb5ad1f51e3cb536560ffe727).  Breaks podman.  I hate it too :(